fleet_vehicle_license_plate_alt/models/fleet_vehicle.py

```diff
@@ -32,4 +32,5 @@ def _name_search(
         rec = self._search(
             expression.AND([domain, args]), limit=limit, access_rights_uid=name_get_uid
         )
-        return models.lazy_name_get(self.browse(rec).with_user(name_get_uid))
+        # return models.lazy_name_get(self.browse(rec).with_user(name_get_uid))
```

Comment:
```suggestion
```

Reply:
Hey, you moved before I say ;-)
As I say, I don't know how to use test, so when Travis set an error on it I tried the first thing that traveled on my mind, and it went!
I hope to have the same luck with next modules.